examples/lerax-square.cpp

```diff
@@ -1,16 +1,30 @@
 #include <iostream>
 #include <cmath>
 #include <string>
+#include <map>
 
 #include <SDL2/SDL.h>
 
 #include "math/Vector2.hpp"
+#include "GameLoop.hpp"
+#include "InputHandler.hpp"
 
 #define SCREEN_WIDTH 800
 #define SCREEN_HEIGHT 600
 
 using namespace std;
 
+// keys to InputHandler observe
```

Comment:
Wouldn't it be better to have this in the EventHandler namespace as a class enum? This would provide the safety (with types) and also be better placed inside a piece of object that is responsible for events.

Reply:
Why and how? That is a specific thing for this example. What you think should be in the enum?

Reply:
This enum should die or be reallocated to internals of hazard-memory. At this time it could be done like that, so we gonna discuss to separate the examples PR from library design PRs in the future.

Reply:
It's not clear to me what would be in the enum. I think this should be better addressed to a particular issue.